tests/install_vm.py

```diff
@@ -245,47 +264,126 @@ def main():
     else:
         data.wait_opt = -1
 
-    # The kernel option 'net.ifnames=0' is used to disable predictable network
-    # interface names, for more details see:
-    # https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/
-    command = 'virt-install --connect={libvirt} --name={domain} --memory={ram} --vcpus={cpu} --network {network} --disk {disk_spec} --initrd-inject={kickstart} --extra-args="inst.ks=file:/{ks_basename} {inst_opt} inst.ks.device=eth0 net.ifnames=0 console=ttyS0,115200" --serial pty --graphics={graphics_opt} --noautoconsole --rng /dev/random --wait={wait_opt} --location={url}'.format(**data.__dict__)
-    if data.uefi == "normal":
-        command = command + " --boot uefi"
-    if data.uefi == "secureboot":
-        command = command + " --boot uefi,loader_secure=yes,\
-loader=/usr/share/edk2/ovmf/OVMF_CODE.secboot.fd,\
-nvram_template=/usr/share/edk2/ovmf/OVMF_VARS.secboot.fd --features smm=on"
+
+def run_virt_install(data):
```

Comment:
To address the CodeClimate report, I think that this function can be further split into a function that forms the command and a function that runs the command.